pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -19,11 +19,11 @@
 from warnings import warn
 import numpy as np
 
-from .lightning_base import LightningBase, LightningBaseFallBack, _chunk_iterable
+from ..core.lightning_base import LightningBase, LightningBaseFallBack, _chunk_iterable
```

Comment:
Does this work, or do we need `from pennylane_lightning.core.lightning_base`

Reply:
This worked just fine in all tests. I can add the full path, though.

Reply:
Done!